Client/src/Components/ListRouter.tsx

```diff
@@ -84,6 +86,7 @@ export const ListRouter = () => {
           <ListItemLink id="profile-menu" to={profileRoute} primary="Profile" icon={<AccountBoxIcon />} />
           <ListItemLink id="about-menu" to={aboutRoute} primary="About Databoom" icon={<InfoIcon />} />
           <ListItemLink id="admin-review" to={adminReviewRoute} primary="Admin Review (#)" icon={<MessageIcon />} />
+          <ListItemLink id="user-review" to={userReviewRoute} primary="Flagged Datasets (#)" icon={<MessageIcon />} />
```

Comment:
whats up with the '(#)'? is it to mimic a red dot?

Reply:
For admin review and I guess normal review, we were supposed to display the number of unapproved datasets. I don't think we've figured out how to do so yet.

Reply:
Yeah it was